src/table/utils/styling-utils.js

```diff
@@ -69,19 +68,22 @@ export const getBaseStyling = (styleObj, objetName, theme) => {
 export function getHeaderStyle(layout, theme) {
   const header = layout.components?.[0]?.header;
   const headerStyle = getBaseStyling(header, 'header', theme);
+  headerStyle.borderWidth = '1px 1px 1px 0px';
 
   // When the table background color from the sense theme is transparent,
   // there is a header background color depending on the header font color
   // to avoid seeing the table body through the table head.
   const headerBackgroundColor = isDarkColor(headerStyle.color) ? '#FAFAFA' : '#323232';
```

Comment:
This one you could move as well right?

Reply:
`headerStyle.borderWidth = '1px 1px 1px 0px';`
and
`contentStyle.borderWidth = '0px 1px 1px 0px';`
so I did not move borderWidth to `getBaseStyling`

Reply:
I mean headerBackgroundColor

Reply:
I had a try, and we need to move both the headerStyle and getBaseStyling then, which it is better to be in styling-utils.js not in table-theme-colors.js

Reply:
came up with a way:
```
const header = (color) => {
    return {
      iconColor: isDarkColor(color) ? 'rgba(255,255,255,0.9)' : 'rgba(0, 0, 0, 0.54)',
      backgroundColor: isDarkColor(color) ? '#FAFAFA' : '#323232',
    };
  };
```
maybe it is a bit overkill. But the idea is to center all those colors.

Reply:
yeah I think it is a bit overkill. I don't like that we add functions to the theme like this. I missed that in this case we are dependent on `getBaseStyling(header, 'header', theme)`, not on the `theme.backgroundColor`

Reply:
It made more sense before

Reply:
sure, i gonna revert it back